ion-c-sys/Cargo.toml

```diff
@@ -5,10 +5,12 @@ edition = "2018"
 
 [dependencies]
 paste = "1.0.0"
+num-bigint = "0.3.0"
 
 [build-dependencies]
-cmake = "0.1"
-bindgen = "0.54"
+cmake = "0.1.44"
+bindgen = "0.54.1"
```

Comment:
I believe this is closer to what we'd want based on [the Cargo book](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html).  The `x.y.z` specifier is basically `>= x.y` and `< x.(y + 1)`,